wiki/content/howto/index.md

```diff
@@ -264,6 +264,50 @@ The debug output also shows UserMeta information:
 - `{item}`: Item posting list
 - `{deleted}`: Delete marker
 
+## Using the Dgraph Sentry Integration
+
+Sentry is a powerful service that allows applications to send arbitrary events, messages, exceptions, bread-crumbs (logs) to your sentry account. In simplest terms, it is a dial-home service but also has a  rich feature set including event filtering, data scrubbing, several SDKs, tagging, environment and release tagging, and integration with Slack, GitHub etc.
+
+Sentry reporting is on by default. However, starting from v20.03.1 and v20.07.0, there is a configuration flag “enable-sentry” which can be used to completely turn off Sentry events reporting. 
```

Comment:
Sentry reporting is on by default. It was first introduced in v20.30.0. However,...